javascripts/tracking.spec.js

```diff
@@ -5,6 +5,19 @@
  * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
  */
 (function () {
+
+    beforeEach(function() {
+        "use strict";
+
+        piwikUsageTracking.targets[0].useAnonymization = true;
+        piwikUsageTracking.targets[1].useAnonymization = false;
+        piwikUsageTracking.targets[2].useAnonymization = false;
```

Comment:
Can be maybe use Anonymization by default for all the trackers in the tests? And then add new tests (maybe even a second tests file or just within a test) where we disable anonymization? It's really important to make sure everything works fine when anonymization is enabled etc. Just to prevent any possible leaks in the future